.gitignore

```diff
@@ -1,3 +1,154 @@
-compose-network/
-db/
-data/
\ No newline at end of file
+## Temporary node files
+network-logs/node/logs/*.log
+network-logs/node/*.log
```

Comment:
We should ignore log files generated anywhere.

```suggestion
*.log
```

Reply:
Agreed. Modified